client/src/components/imagine25/components/Frame.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ALLButton from "src/components/all-components/ALLButton";
+
+export const Frame = (content, nextOnClick) => {
+  //Constant syling methods
+  const blueLine =
+    "tw-w-[35vw] tw-h-[5px] tw-bg-[#0045d5] tw-my-[15px] tw-mx-auto";
+  const yellowLine =
+    "tw-w-[5px] xs:tw-h-[325px] sm:tw-h-[350px] md:tw-h-[350px] xl:tw-h-[400px] 2xl:tw-h-[425px] tw-bg-[#ffc335] tw-my-[15px] tw-mx-auto";
+  return (
+    <>
+      <div className={blueLine}></div>
+      <div className="d-flex justify-content-center">
+        <div className={yellowLine}></div>
+        <div>{content}</div>
+        <div className={yellowLine}></div>
+      </div>
+      <div className={blueLine}></div>
+      <div className="tw-flex tw-justify-center tw-space-x-[35vw] my-4">
```

Comment:
nit: add some more padding above the previous and next buttons. maybe add a `tw-pt-14`